spec/jobs/create_institution_records_job_spec.rb

```diff
@@ -5,9 +5,11 @@
 RSpec.describe CreateInstitutionRecordsJob, type: :job do
   describe 'after enqueuing the job' do
     it 'adds the institutions from the RSPO API to the database from a given type' do
+      WebMock.allow_net_connect!
       CreateInstitutionTypesJob.new.perform
       CreateInstitutionRecordsJob.new.perform(93)
       expect(Institution.where(rspo_institution_type_id: 93).count).to be_positive
+      WebMock.disable_net_connect!
```

Comment:
This can keep net connections enabled if an error occurs between allow and disable. As an alternative, you could add:
```ruby
RSpec.configure do |config|
  config.around(:each, allow_net_connect: true) do |example|
    WebMock.allow_net_connect!
    example.run
  ensure
    WebMock.disable_net_connect!
  end
end
```
to your rails_helper and then add a tag to your test
```ruby
it '...', :allow_net_connect do
  # here net connections will be enabled
end
```